### ISSUE TITLE
Handle per-hemisphere surface files

### ISSUE THREAD

**Issue by rmarkello** [ME-ICA/tedana#33]
As a result of merging #22, running 

```
tedana -d sub-X_echo-[1,2,3].[L,R].func.gii -e 15.0 30.0 45.0
```

will cause the program to fail. The provided data would be interpreted as six independent echo files, even though it should only be three echos separated by hemisphere.

The `tedana.utils.utils.load_data()` function could be updated to handle this, but there needs to be some discussion about how users should supply surface data. That is, should there be requirement to supply data in hemisphere-major or echo-major order? Parsing filenames would require enforcing a specific naming convention, and parsing metadata would presume that the files will have appropriate metadata. 

I am open to suggestions, but this bug currently prevents providing L and R hemisphere surface data together.

**Comment by emdupre** [comment]
Well, users could supply all six files individually at the command line, no ? But that's 👎 

Personally I'd prefer to parse metadata, though this is coming more from a BIDS perspective. This is still assuming a slightly broader issue is solved, which is the spatial localization with surface data.

**Comment by rmarkello** [comment]
Even if users supplied all the files individually, `load_data()` would still need to have a mechanism to say which were different echos versus hemispheres. In the example I provided, assuming you ran it from a bash shell, it would get expanded to echo-major order. But a user could specify them in hemisphere-major order on the command line just as easily...

As for metadata: while there are a significant number of standardized metadata fields for e.g., GIFTI files, those fields aren't necessarily populated by the software that produce the files... One option is providing a command-line argument specifying the order (e.g., `--surf-order echo` vs `--surf-order hemisphere`)? That way it gets left up to users to supply their data properly?

**Comment by emdupre** [comment]
I went ahead and closed this with #63, although we'll need to re-open this if we decide to move forward with supporting gifti files with multiple naming conventions (i.e., beyond `*.[L,R].func.gii`). Hopefully, this kind of file handling will be obsolete once `tedana` is better integrated into FMRIPrep and afni_proc.py via #26.
